src/pages/verify/integrations/firebase.mdx

```diff
@@ -0,0 +1,120 @@
+
+1. Create a new Firebase project or use an existing one.
+2. Enable OpenID Connect as a sign-in provider for your Firebase project.
+   You can do this by going to the Authentication section of your Firebase console and choosing the Sign-in method tab.
+   Click the "Add provider" button and select "OpenID Connect".
+   ![OIDC custom provider](./firebase/OIDC-custom-provider.png)
+3. You will now enable and define a new OIDC provider using the [information from your Criipto Dashboard and the Client Secret you previously saved](#firebase-application-and-oidc-provider-setup).
+4. Choose the grant type you intend to use. We recommend choosing the OAuth2 Code Flow.
+5. Enter a provider name (in this example we used "Criipto").
+6. Enter your Criipto Application client id in the "Client ID" field.
+7. Enter `https://{{YOUR_CRIIPTO_DOMAIN}}` in the "Issuer (URL)" field.
+8. Enter your Criipto Application client secret in the "Client Secret" field and click "Next".
+   ![Define new OIDC provide](./firebase/define-new-OIDC-provider.png)
+9. You will now configure the OIDC integration. Copy the Firebase Callback URL and add it to the "Callback URL" field in your Criipto Dashboard.
+   ![Callback URL in Criipto Dashboard](./firebase/criipto-dashboard-callback-url.png)
+10. You are now ready to integrate Firebase into your application by following the steps for the platform of your choice:
+
+    - [Web applications](https://firebase.google.com/docs/auth/web/openid-connect)
+    - [iOS](https://firebase.google.com/docs/auth/ios/openid-connect)
+    - [Android](https://firebase.google.com/docs/auth/android/openid-connect)
+
+You can handle the entire sign-in flow with the Firebase SDK, or choose to handle it manually. The latter option is useful if you've already implemented one of Criipto's SDKs in your application.
+
+### Manual setup for handling the sign-in flow
+
+As mentioned above, you can choose the [manual setup](https://firebase.google.com/docs/auth/web/openid-connect#manual) and use the ID token directly to authenticate with Firebase.
+
+In `@criipto/verify-react`, you can use the `id_token` property of the `result` object to retrieve the ID token after the user signed in with Criipto Verify.
+
+```jsx
+import { getAuth, signInWithCredential, OAuthProvider } from 'firebase/auth';
+
+const provider = new OAuthProvider('oidc.criipto');
+
+if (claims) {
+  const credential = provider.credential({
+    idToken: result.id_token,
+  });
+
+  signInWithCredential(getAuth(), credential)
+    .then((result) => {
+      // User is signed in.
+      // IdP data available in result.additionalUserInfo.profile.
+
+      // Get the OAuth access token and ID Token
+      const credential = OAuthProvider.credentialFromResult(result);
+      const idToken = credential.idToken;
+    })
+    .catch((error) => {
+      // Handle error.
+    });
+}
+```
+
+With `@criipto/auth-js`, you can use the [match method](https://github.com/criipto/criipto-auth.js#match):
```

Comment:
Suggestion to expand this to mention redirect authorization with the library, since that is the case where #match works.
Perhaps something along the lines of:

Using `@criipto/auth-js` with redirect authorization you can use the resulting `id_token` from the `match()` call to pass to Firebase.